README.md

```diff
@@ -96,14 +96,45 @@ You can find those example applications under `examples/`.
 
 ### Run the Trace Server
 
-In order to open traces, you need a trace server running on the same machine as the trace extension. You can download the [Eclipse Trace Compass server][tc-server] or let `yarn` download and run it:
+In order to open traces, you need a trace server running on the same machine as the trace extension. There are 3 options to run the trace server. You can build and start the server manually using the server development environment, from the command line, or from the Theia application. The latter two options requires you to download a Trace Server build. See [Getting a Trace Server build][tc-server-download-guide] for more details.
+
+#### Building with the server development environment
+
+You can build and run the trace server manually using Eclipse. Starting it manually, has the advantage, that the open-traces
+are populated with traces on the server, or the server can be started in debug mode from the server development environment to debug the server. To set up the development environment for the Trace Server, see [here][tc-server-build].
+
+Please note that you need to stop the server manually as well.
+
+#### Starting the Trace Server from the command line
+
+To start the Trace Server from the command line, use the following command:
 
 ```bash
-yarn download:server
 yarn start:server
 ```
 
-You can also build the trace-server yourself using Trace Compass and the Incubator. Take a look at the [instructions here][tc-server-build].
+Please note that you need a Trace Server Build before starting the server. See [Getting a Trace Server build][tc-server-download-guide] for more details. You will need to stop the server manually as well.
+
+#### Starting the Trace Server from the Theia application
```

Comment:
In this chapter you should describe the preference setting for the trace server. User can change it to point to point to a custom trace server.